Python/Product/PythonTools/visualstudio_py_util.py

```diff
@@ -134,7 +134,9 @@ def exec_module(module, global_variables):
 def read_bytes(conn, count):
     b = to_bytes('')
     while len(b) < count:
-        b += conn.recv(count - len(b))
+        received_data = conn.recv(count - len(b))
+        if received_data:
+            b += received_data
```

Comment:
Receiving no data typically indicates the end of the stream - is looping the right behavior here or should we break? @int19h may have some idea.